src/Blogger.Infrastructure/Services/Externals/EmailService.cs

```diff
@@ -1,12 +1,31 @@
 ﻿using Blogger.Application.ApplicationServices;
+using Microsoft.Extensions.Options;
+using System.Net;
+using System.Net.Mail;
 
 namespace Blogger.Infrastructure.Services.Externals;
 
-public class EmailService : IEmailService
+public class EmailService(IOptions<EmailSettings> options) : IEmailService
 {
-    public Task SendAsync(string email, string subject, string content, CancellationToken cancellationToken)
+    private readonly EmailSettings _emailSettings = options.Value;
+
+    public async Task SendAsync(string email, string subject, string content, CancellationToken cancellationToken)
     {
-        
-        return Task.CompletedTask;
+        using var smtpClient = new SmtpClient(_emailSettings.SmtpHost, _emailSettings.SmtpPort)
+        {
+            EnableSsl = true,
+            Credentials = new NetworkCredential(_emailSettings.From, _emailSettings.AppPassword)
+        };
+
+        var mailMessage = new MailMessage
+        {
+            From = new(_emailSettings.From),
+            Subject = subject,
+            Body = content
```

Comment:
It's better to set encoding for the body and subject to support all languages:
```csharp
mailMessage.SubjectEncoding = System.Text.Encoding.UTF8;
mailMessage.BodyEncoding =  System.Text.Encoding.UTF8;
```
Also consider using HTML for the body as well, it may be helpful:
```csharp
mailMessage.IsBodyHtml = true;
```

Reply:
Thanks for mentioning this key points 🙏